bellman/src/groth16/mod.rs

```diff
@@ -301,7 +301,7 @@ impl<E: Engine> Parameters<E> {
                 .into_affine_unchecked()
             }
             .map_err(|e| io::Error::new(io::ErrorKind::InvalidData, e))
-            .and_then(|e| if e.is_zero() {
+            .and_then(|e| if /*e.is_zero()*/false {
```

Comment:
Necessary to save the parameters with `should_filter_points_at_infinity == false` as far as I understand.

Reply:
I think it would still be good to have it as a parameter - normal Bellman doesn't need the points at infinity, it's just circom.

Reply:
The point itself is definitely valid - in our procedure, you need these zero points.